gazebo_plugins/CMakeLists.txt

```diff
@@ -135,7 +129,6 @@ catkin_package(
   camera_info_manager
   std_msgs
   DEPENDS 
-    gazebo 
```

Comment:
Since all CMake variables are case sensitive I don't think it would be good to make anything in `catkin` case insensitive.

`catkin` is supposed to already check the arguments and ensure the passed name as been found before. I tried this specific example and the condition checking this was incorrect. ros/catkin#813 addresses this and with it applied the above CMake will result in the intendend error message that `gazebo` was not found (because `gazebo_FOUND` is not set).